test/expected/slim.css

```diff
@@ -3584,10 +3581,6 @@ hr {
         -ms-flex-direction: column;
             flex-direction: column;
   }
-.flex-row-sm-wrap {
```

Comment:
do we use this anywhere?

Reply:
Yeah we do but as this is a breaking release, better now than later. We don't rely on the fact that it adds the negative margin though. But we can't use `display-flex` in conjunction with that right now since it makes things get negative margins

Reply:
hah - it was a breaking release - we are using it in production hah

Reply:
It should only be a matter of replacing those classes:

`flex-row-wrap` -> `flex-wrap`
`flex-row-sm-wrap` -> `flex-sm-wrap`

I could maintain the name and remove the side effects, seemed misleading though.